src/main/java/com/videoclub/suriken/model/Movie.java

```diff
@@ -0,0 +1,148 @@
+package com.videoclub.suriken.model;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+
+@Entity
+public class Movie {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    private Long movId;
+
+    private String movName;
+
+    private int movYear;
+
+    @Enumerated(EnumType.STRING)
+    private Genre movGenre;
+
+    private String movDirector;
+
+    private int stock;
+
+    @ManyToMany()
+    private List<MovieRenter> movieRenters = new ArrayList<>();
+
+    public Movie(Long movId,
+                 String movName,
+                 int movYear,
+                 Genre movGenre,
+                 String movDirector,
+                 int stock,
+                 List<MovieRenter> movieRenters) {
+        this.movId = movId;
+        this.movName = movName;
+        this.movYear = movYear;
+        this.movGenre = movGenre;
+        this.movDirector = movDirector;
+        this.stock = stock;
+        this.movieRenters = movieRenters;
+    }
+
+    public Movie() {
+    }
+
+    public void addMovieRenter(MovieRenter movieRenter) {
+        movieRenters.add(movieRenter);
```

Comment:
Ovo je greska u mom kodu, tj. nisam verovao da ce hibernate umeti da razresi obe tabele. Procitaj ovo: 
- https://vladmihalcea.com/a-beginners-guide-to-jpa-and-hibernate-cascade-types/
- https://howtodoinjava.com/hibernate/hibernate-jpa-cascade-types/

To isto moze da bude na interview
A i koriste se apsolutno puno, mora da se zna